README.md

```diff
@@ -3,41 +3,93 @@
 
-## About
+### What is TC2 *not*?
 
-What is Team Comtress 2? It's a version of Team Fortress 2, based on the recent leak which aims to fix many bugs, performance issues, etc. Imagine it like [mastercomfig](https://mastercomfig.com/) super charged!
+* **TC2 is *not* a "pro mod"**
 
-Obviously, as a leaked build, it's not useful for getting better performance in Casual on its own (you can't use this build to connect to any existing servers), but it can help me a lot if you all can test it, so that I am more confident in sending many of these changes to Valve for them to include in the base game! Please let me know how it works for you!
+While much of the work in TC2 will improve the competitive experience, TC2 does not touch gameplay, balance, or competitive rules.
 
-## Install
+* **TC2 is *not* a vision for the "good old days" of TF2**
 
-1. [Download](https://github.com/mastercomfig/team-comtress-2/releases/latest) `game_clean.zip` from the latest release.
-2. Copy your current `Team Fortress 2` installation to a new folder.
-3. Extract the ZIP download to this new folder.
-4. Make sure you don't have any configs installed.
-5. Double click `start_tf2.bat`. Note that you must have Steam running.
-6. Enjoy!
+TC2 will not feature game design changes, gameplay additions, or almost any form of new content. If you're looking for this kind of content, [Team Fortress 2 Classic](https://tf2classic.com/) is more up your alley.
 
-Note: TC2 bundles the stock HUD with [CriticalFlaw's HUD fix](https://github.com/CriticalFlaw/TF2-HUD-Fix) in the custom folder
+* **TC2 is *not* the modern game, with new content/updates**
+
+You're looking for [Creators.TF](https://creators.tf/).
+
+* **TC2 is *not* an alternative, more stable game experience**
+
+TC2 is based on the 2017 TF2 source code leak, which contained an in-development version of Jungle Inferno. This build is rife with issues and incomplete features, and *is not compatible with* the modern game client.
+
+## How can I help?
+
+1. **Report bugs that exist in *both* TF2 *and* TC2 through the [issues page](https://github.com/mastercomfig/team-comtress-2/issues).**
+	* Bugs that are present in modern TF2 but are not present in TC2 cannot be fixed in this project.
+2. **Suggest quality of life improvements (*not balance changes*) through the [issues page](https://github.com/mastercomfig/team-comtress-2/issues).**
+	* These changes should not affect gameplay, and should be unanimously desired.
+3. **Install TC2 and play the game to test stability, bugfixes, and performance.**
+	* Information about multiplayer playtests can be found [on our Discord server](https://discord.gg/CuPb2zV).
+
+## Installation
+
+### Basic Install Instructions
+
+New users should follow this approach.
+
+*Note: TC2 is Windows-only (for now; see [#2](https://github.com/mastercomfig/team-comtress-2/issues/2))*

```

Comment:
You still can play in Wine so I would change it to "TC2 currently has only Windows binaries" or something

Reply:
the linux build stuff is getting sorted out soon anyway so I'm not going to bother, we'll leave the windows-only blurb until the linux PR adds linux documentation to the readme. the sooner we get this pushed out, sooner the linux guys can work from *this* readme instead of the old one